package.json

```diff
@@ -15,10 +15,14 @@
   "engines": {
     "node": ">=6.11.5"
   },
+  "private": true,
```

Comment:
This would'nt cause lerna to not publish this package? Or if i misunderstood the [docs](https://github.com/webpack/webpack-cli.git) it won't publish just packages inside `packages/`?

Reply:
Yeah that's true, we must update them